### ISSUE TITLE
Scrollable TabController within a ScrollVIew with stickyHeaderIndices

### ISSUE THREAD

**Issue by lumotroph** [wix/react-native-ui-lib#1031]
Thanks for the fantastic work, everyone!

Any suggestions on how to get a scrollable view to work with your great `TabController` component and still have the contents of the `TabPage` scroll?

To explain:
We have a `ScrollView` with some items and then a `TabController`, each `TabPage` under the controller has a `FlatList` of items. When we set the `stickyHeaderIndices` prop on the `ScrollView` to the correct component (the index of our `TabController`), we get correct sticky-ness, but the items in the `FlatList` are no longer scrollable.

In other words:
Setting `stickyHeaderIndices` to the correct component leads to a situation where scrolling works, the TabBar (or "tab header") is sticky, but the content within the rest of the component does not scroll.

Any ideas?

**Comment by ethanshar** [comment]
Hi @lumotroph 
It would really help if you'll add some example.
We'll be able to understand better the issue and investigate..
